### DISCUSSION TITLE
1.4.3

### DISCUSSION THREAD

**Discussion by ilyhalight** [FOSWLY/vot-cli#36] [General]
## Список изменений

- Добавлена поддержка загрузки субтитров в `.srt` (#33)

## What's Changed
* json subtitles to srt  format by @pgorun in https://github.com/FOSWLY/vot-cli/pull/33

## New Contributors
* @pgorun made their first contribution in https://github.com/FOSWLY/vot-cli/pull/33

**Full Changelog**: https://github.com/FOSWLY/vot-cli/compare/1.4.2...1.4.3

<hr /><em>This discussion was created from the release <a href='https://github.com/FOSWLY/vot-cli/releases/tag/1.4.3'>1.4.3</a>.</em>

**Comment by AlexGreen87** [comment]
Привет. Перестал скачивать перевод с **ВК Видео**, так как у него сменился адрес на **vkvideo.ru**. Есть возможность добавить?

**Reply by ilyhalight** [reply]
Пока используйте через старый домен, через него должно переводить. Например, вместо `https://vkvideo.ru/video-197905200_456239163`, пишите `https://vk.com/video?z=video-197905200_456239163`

Новый домен будет в следующем обновление, которое надеюсь выйдет до конца года

**Comment by vukitoso** [comment]
если установить версию 1.4.3 и запустить с параметром --version, то программа покажет 1.4.2
